bolt/metrics.go

```diff
@@ -75,10 +84,43 @@ func (c *Client) Describe(ch chan<- *prometheus.Desc) {
 	ch <- dashboardsDesc
 	ch <- scrapersDesc
 	ch <- telegrafsDesc
+	ch <- telegrafPluginsDesc
 	ch <- boltWritesDesc
 	ch <- boltReadsDesc
 }
 
+type instaTicker struct {
+	tick   chan struct{}
+	timeCh <-chan time.Time
+}
+
+var (
+	// ticker is this influx' timer for when to renew the cache of configured plugin metrics.
+	ticker *instaTicker
+	// telegrafPlugins is a cache of this influx' metrics of configured plugins.
+	telegrafPlugins = map[string]float64{}
+)
+
+// Initialize a simple channel that will instantly "tick",
+// backed by a time.Ticker's channel.
+func init() {
+	ticker = &instaTicker{
+		tick:   make(chan struct{}, 1),
+		timeCh: time.Tick(time.Minute * 59),
+	}
+
+	ticker.tick <- struct{}{}
+
+	go func() {
+		for {
+			select {
+			case <-ticker.timeCh:
```

Comment:
why not just use `for t := range ticker.Timech`?  Curious why this is here in bolt and not kv? Are these not intended for Cloud?

Reply:
This is the bolt implementation (`bolt/*.go`) as cloud reports metrics differently. The important "shared" logic is in `kv/telegraf.go`. A cloud PR will be forthcoming that utilizes the data stored from that shared logic and reports it as well.